src/logger/logger.ts

```diff
@@ -4,6 +4,13 @@ import * as uuid from 'uuid';
 import stringify from 'fast-safe-stringify';
 import isError from 'is-error';
 
+export const redactSecret = (data: any): any => {
```

Comment:
1. Now that we're sharing the function in the `OpenApiWrapper` it would make sense to move it out to `util.ts` so that the  `OpenApiWrapper` doesn't have a direct dependency on `Logger`
2. Thanks to this being a public function, you can write some more unit tests :D The one I'm curious about is what happens, when you pass JS object as `data`. E.g. `{someKey: { someNestedKey: "someNestedValue" }} ... can you call `.replace` on that?

I'm specifically asking because we use it as
```
redactSecret(err.config.data);
```
which means we expect the `err.config.data` to be always a string. If it wouldn't be string, then we cannot call `replace` anymore, because such function doesn't exist on any random object, right?